class_names: ["power line", "pole", "tower"]
model_path: "/home/bofia/powerGrid_ws/src/det_pkg/models/best.onnx"
confidence_threshold: 0.3
iou_threshold: 0.3
nms_sigma: 0.5
input_shape: [1, 3, 640, 640]
cuda_enable: true
intra_opNum_threads: 1
log_severity_level: 3